sirepo/template/ml.py

```diff
@@ -736,6 +744,43 @@ def _get_fit_report(report, x_vals, y_vals):
     return param_vals, param_sigmas, plots
 
 
+def _get_remote_data(url, headers_only):
+    import os
+    import urllib
+    from urllib import parse
+    from urllib import request
+
+    filename = os.path.basename(urllib.parse.urlparse(url).path)
+    try:
+        with urllib.request.urlopen(url) as r:
+            if headers_only:
+                return PKDict(headers=_header_str_to_dict(r.headers))
+            chunk_size = 1024 * 1024
```

Comment:
This is a constant. Move out off the try and possibly global scope.

Reply:
[9567d25](https://github.com/radiasoft/sirepo/pull/4671/commits/9567d2543e869bedeebbd4ed87255d8e96e64a96)

(is a mibibyte reasonable?)